airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -120,6 +118,7 @@
   private static final int STDIN_REMOTE_PORT = 9001;
   private static final Map<String, String> AIRBYTE_POD_LABELS = Map.of("airbyte", "worker-pod");
```

Comment:
we forgot to do this when we added the command fetcher pod, can we add a note in the class docstring about the assumption that images need an AIRBYTE_ENTRYPOINT env var set?